barretenberg/cpp/src/barretenberg/relations/ultra_arithmetic_relation.hpp

```diff
@@ -12,6 +12,12 @@ template <typename FF_> class UltraArithmeticRelationImpl {
         5  // secondary arithmetic sub-relation
     };
 
+    /**
+     * @brief Total degrees of sub-relations considered as polynomials in witnesses.
```

Comment:
```suggestion
     * @brief Total degrees of subrelations considered as polynomials in witnesses.
```

Reply:
Could you rephrase this comment so it's clear where it's being used? Also I think it's quite a confusing comment. Here and in the other places